src/main/java/net/coderbot/iris/shaderpack/transform/Transformations.java

```diff
@@ -11,6 +11,7 @@
 	enum InjectionPoint {
 		DEFINES,
 		BEFORE_CODE,
+        MAIN_TAIL,
```

Comment:
Wrong formatting

Reply:
Fixed. (And im gonna blame this on intellij which displayed it properly)